### PR TITLE
How to debug standard lightning components

### PR DESCRIPTION
**Pull request by SantuCodeX**
Ticket included : [AVEBLOG__0051](https://theavenoir.atlassian.net/browse/AVEBLOG-68)
Issue : How to debug standard lightning components
Cause : N/A
fix: N/A

### CODE REVIEW

File: force-app/main/default/lwc/debugLightningCmp/debugLightningCmp.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            this.error = error;
+        }
+    }
+
+    //Edit form 
+    
+    handleSubmit(event) {
+        event.preventDefault();      
+        const fields = event.detail.fields;
+        console.log(JSON.stringify(fields));
```
Comment (nikhilmehra2906) [inline force-app/main/default/lwc/debugLightningCmp/debugLightningCmp.js:41]: Remove all console log statements

File: force-app/main/default/lwc/debugLightningCmp/debugLightningCmp.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+const FIELDS = ["Contact.Name", "Contact.Phone","contact.Email"];
+
+export default class DebugLightningCmp extends LightningElement {
+
+    recordId = "003H4000001jJqiIAE";
+    contactData;
+    error;
+
+
```
Comment (nikhilmehra2906) [inline force-app/main/default/lwc/debugLightningCmp/debugLightningCmp.js:24]: Remove spaces

File: force-app/main/default/lwc/debugLightningCmp/debugLightningCmp.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            console.log(JSON.stringify(data));
+            this.contactData = data;
+        }else if(error) {
+            console.log(JSON.stringify(error));
+            this.error = error;
+        }
+    }
+
+    //Edit form 
+
```
Comment (nikhilmehra2906) [inline force-app/main/default/lwc/debugLightningCmp/debugLightningCmp.js:37]: remove space

File: force-app/main/default/lwc/debugLightningCmp/debugLightningCmp.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+***********************************************************************/
+
+import { LightningElement,wire } from 'lwc';
+import { getRecord } from "lightning/uiRecordApi"
+
+const FIELDS = ["Contact.Name", "Contact.Phone","contact.Email"];
+
+export default class DebugLightningCmp extends LightningElement {
+
+    recordId = "003H4000001jJqiIAE";
```
Comment (nikhilmehra2906) [inline force-app/main/default/lwc/debugLightningCmp/debugLightningCmp.js:20]: Do not hardcode Id use @api

File: force-app/main/default/lwc/debugLightningCmp/debugLightningCmp.html [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    <template if:true={contactData}>
+        <ul>
+            <li>{contactData.fields.Name.value}</li>
+            <li>{contactData.fields.Phone.value}</li>
+            <li>{contactData.fields.Email.value}</li>
+        </ul>
+    </template>
+</lightning-card>
+    <lightning-card title="Edit">
+
```
Comment (nikhilmehra2906) [inline force-app/main/default/lwc/debugLightningCmp/debugLightningCmp.html:12]: remove space

File: force-app/main/default/lwc/debugLightningCmp/debugLightningCmp.js (hunk 1 above)
Comment (SantuCodeX) [reply force-app/main/default/lwc/debugLightningCmp/debugLightningCmp.js:41]: This is used to show how we can use console.log for debugging lwc component

File: force-app/main/default/lwc/debugLightningCmp/debugLightningCmp.js (hunk 4 above)
Comment (SantuCodeX) [reply force-app/main/default/lwc/debugLightningCmp/debugLightningCmp.js:20]: This is just for testing purpose and how we can debug component I am not using any parent child

**Pull request merged**